settings.gradle

```diff
@@ -1 +1,2 @@
 include 'acmepos', 'kebabpos', 'motelpos', 'tablepos', 'utils'
+include 'ramenpos'
```

Comment:
Why is it a separate statement? It should go in with the rest of them.

Reply:
IDE does it automatically